conf/application.conf

```diff
@@ -102,7 +102,7 @@ metrics {
 # Microservice specific config
 
 mongodb {
-  uri = "mongodb://localhost:27017/eori-common-component-frontend"
+  uri = "mongodb://localhost:27017/cf"
```

Comment:
Non-blocking q: Is this the name of your local database?

Reply:
No. Must have changed this by accident. Surprised it still works.. Thx :)